include/fmt/base.h

```diff
@@ -149,6 +149,13 @@ import std;
 #  define FMT_USE_NONTYPE_TEMPLATE_ARGS 0
 #endif
 
+// Detect C++20 concepts
+#ifdef __cpp_concepts
+#  define FMT_USE_CONCEPTS 1
+#else
+#  define FMT_USE_CONCEPTS 0
+#endif
```

Comment:
The usual pattern is to first check if `FMT_USE_*` is provided by the user, see e.g. `FMT_USE_NONTYPE_TEMPLATE_ARGS`. Let's do the same here.